### DISCUSSION TITLE
Looking for workaround - Webhoster cronjob time out

### DISCUSSION THREAD

**Discussion by Randalix** [FreshRSS/FreshRSS#5597] [Q&A]
Hi!
Problem:
Refresh cronjob times out, because my webhoster limits cronjobs to take 120s and only once per hour.

What I have tried:
- Remove dead feeds in order to make the refresh faster. (still takes too long)
- Tried the auto update plugin, but could not get it working. (Is it in a working state?)

My idea for a workaround:
- Locally scheduled script to trigger the refresh. Would need to figure out the authentication for that.

Has someone suggestions what I could try? Or can you give me tips on how to get the local refresh script going?

Thanks in advance!

Btw: Fresh rss is soo cool! Really love the project <3

**Comment by Alwaysin** [comment]
You could try to activate the "authentication token" and call https://freshrss.example.com/i/?c=feed&a=actualize&ajax=1&user=***&token=*** from somewhere else.

See https://github.com/FreshRSS/FreshRSS/blob/edge/docs/en/users/09_refreshing_feeds.md#for-form-authentication (screenshot is not up to date, you can find the conf in profile management)

**Comment by Randalix** [comment]
Thanks for your reply. For now I used  this script as a systemd timer, cron will work as well of course (sorry forgot to update the thread):
curl -X POST "https://mysite.com/freshrss/p/i/?c=feed&a=actualize" -u "user:password"
